xradar/io/backends/common.py

```diff
@@ -57,7 +57,8 @@ def _fix_angle(da):
 def _attach_sweep_groups(dtree, sweeps):
     """Attach sweep groups to DataTree."""
     for i, sw in enumerate(sweeps):
-        DataTree(sw, name=f"sweep_{i}", parent=dtree)
+        dtree[f"/sweep_{i:02d}"] = sw
```

Comment:
@kmuehlbauer and @mgrover1, do you think it's a good idea to have sweep_00, sweep_01, and so on? I proposed this change because saving and reopening a `DataTree` with more than ten nodes will load them in a different order. This is just to keep the nodes ordered.

Reply:
@aladinor Unfortunately this isn't going to work since this is enforced by the datamodel. If this can't be fixed upstream, we'll need to think about some clever method to fix it on our side.

Reply:
@kmuehlbauer, the issue relies on when opening a datatree. Lets look a the data we used during the ERAD short course. 

```python 
import s3fs
import xarray as xr
import fsspec

URL = 'https://js2.jetstream-cloud.org:8001/'
path = f'pythia/radar/erad2024'
fs = s3fs.S3FileSystem(anon=True, client_kwargs=dict(endpoint_url=URL))
file =  s3fs.S3Map(f"{path}/zarr_radar/erad_2024.zarr", s3=fs)

dtree = xr.backends.api.open_datatree(
    file, 
    engine='zarr', 
    consolidated=True, 
    chunks={}
)
print(dtree)
```
Thus the expected output will be `sweep_0`, `sweep_1`, ..., `sweep_20` ordered in an ascent way. However, when we use the `open_datatree` the nodes will be ordered `sweep_0`, `sweep_1`, `sweep_11`, `sweep_11`, ..., `sweep_2`, `sweep_20`. 

Probably, this is a manner of visualization, but I think it could be fixed in that way

```python 

<xarray.DataTree 'cband'>
Group: /cband
├── Group: /cband/sweep_0
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_1
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_10
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_11
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
.
.
.
.

├── Group: /cband/sweep_2
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_3
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_4
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_5
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_6
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_7
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
├── Group: /cband/sweep_8
│       Dimensions:                              (volume_time: 73, azimuth: 360,
│                                                 range: 492)
│       Coordinates:
│           altitude                             float32 4B ...
│         * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
│           crs_wkt                              int64 8B ...
│           elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│           latitude                             float32 4B ...
│           longitude                            float32 4B ...
│         * range                                (range) float32 2kB 250.0 ... 2.457e+05
│           time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
│         * volume_time                          (volume_time) datetime64[ns] 584B 20...
│           x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
│           z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
│       Data variables: (12/14)
│           differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
│           reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           ...                                   ...
│           sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
│           sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
│           uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
│           velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
└── Group: /cband/sweep_9
        Dimensions:                              (volume_time: 73, azimuth: 360,
                                                  range: 492)
        Coordinates:
            altitude                             float32 4B ...
          * azimuth                              (azimuth) float32 1kB 0.5 1.5 ... 359.5
            crs_wkt                              int64 8B ...
            elevation                            (volume_time, azimuth) float32 105kB dask.array<chunksize=(73, 360), meta=np.ndarray>
            latitude                             float32 4B ...
            longitude                            float32 4B ...
          * range                                (range) float32 2kB 250.0 ... 2.457e+05
            time                                 (volume_time, azimuth) datetime64[ns] 210kB dask.array<chunksize=(73, 360), meta=np.ndarray>
          * volume_time                          (volume_time) datetime64[ns] 584B 20...
            x                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
            y                                    (azimuth, range, volume_time) float32 52MB dask.array<chunksize=(90, 123, 19), meta=np.ndarray>
            z                                    (volume_time, azimuth, range) float32 52MB dask.array<chunksize=(19, 90, 123), meta=np.ndarray>
        Data variables: (12/14)
            differential_reflectivity            (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            nyquist_velocity                     (volume_time, azimuth) float32 105kB dask.array<chunksize=(1, 360), meta=np.ndarray>
            pulse_width                          (volume_time, azimuth) timedelta64[ns] 210kB dask.array<chunksize=(1, 360), meta=np.ndarray>
            reflectivity                         (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            reflectivity_hh_clut                 (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            reflectivity_vv                      (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            ...                                   ...
            sweep_fixed_angle                    (volume_time) float32 292B dask.array<chunksize=(73,), meta=np.ndarray>
            sweep_mode                           (volume_time) <U20 6kB dask.array<chunksize=(73,), meta=np.ndarray>
            sweep_number                         (volume_time) int64 584B dask.array<chunksize=(73,), meta=np.ndarray>
            uncorrected_cross_correlation_ratio  (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            uncorrected_differential_phase       (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>
            velocity                             (volume_time, azimuth, range) float64 103MB dask.array<chunksize=(1, 360, 492), meta=np.ndarray>

```

Reply:
@aladinor Maybe it's really a matter of the repr? I'll experiment a bit tomorrow.

The _MN approach fixes this for number of sweeps <=100. But if someone wants to create a volume of >100 sweeps? Not very likely? That's 6 seconds per rotation over 10 Minutes. It would be possible... : grinning:

Reply:
Well, I'm not sure, but who knows...  I think we can tackle this in another way.